ingress/controllers/nginx/nginx.tmpl

```diff
@@ -155,6 +155,7 @@ http {
     server {
         listen 80{{ if $cfg.useProxyProtocol }} proxy_protocol{{ end }};
         {{ if $server.SSL }}listen 443{{ if $cfg.useProxyProtocol }} proxy_protocol{{ end }} ssl http2;
+        # PEM sha: {{ $server.SSLPemChecksum }}
```

Comment:
Is just a comment. If the secret changes this comment will trigger the reload (diff with the current running cfg)